server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -398,12 +411,14 @@ def main():
             # Rewind to the beginning.
             reportfp.seek(0)
 
+            logger.debug('Sending report: start')
             report = Report(config, _NAME_)
             report.init_report_template()
             try:
                 report.post_status(config.timestamp(), "status", reportfp.name)
             except Exception:
                 pass
+            logger.debug('Sending report: end')
```

Comment:
We have added a lot of debug statements.  Is debug the default logging level by default?

Reply:
Yes, it is. We probably want it at DEBUG for unit tests and staging, but we can turn it down to INFO for production. One possibility is to specify it in the config file (a general default and perhaps individual defaults for each program).